ConsentViewController/Classes/Consents/SPCCPAConsent.swift

```diff
@@ -144,25 +177,18 @@ protocol CampaignConsent {
         status = try container.decode(CCPAConsentStatus.self, forKey: .status)
         rejectedVendors = try container.decode([String].self, forKey: .rejectedVendors)
         rejectedCategories = try container.decode([String].self, forKey: .rejectedCategories)
-        uspstring = try container.decode(SPUsPrivacyString.self, forKey: .uspstring)
+        signedLspa = try container.decode(Bool.self, forKey: .signedLspa)
         uuid = try container.decodeIfPresent(String.self, forKey: .uuid)
         childPmId = try container.decodeIfPresent(String.self, forKey: .childPmId)
         consentStatus = try (try? container.decode(ConsentStatus.self, forKey: .consentStatus)) ?? ConsentStatus(from: decoder)
         webConsentPayload = try container.decodeIfPresent(SPWebConsentPayload.self, forKey: .webConsentPayload)
     }
 
-    public static func rejectedNone () -> SPCCPAConsent { SPCCPAConsent(
```

Comment:
what happened to `rejectedNone`? Are you sure we no longer need this method?

Reply:
`rejectNone` was probably used in the past, but there's no occurrence of it in SDK 7